src/OSPSuite.InstallationValidator.Core/Services/BatchStarterTask.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.IO;
+using OSPSuite.InstallationValidator.Core.Assets;
+using OSPSuite.InstallationValidator.Core.Domain;
+using OSPSuite.Utility;
+
+namespace OSPSuite.InstallationValidator.Core.Services
+{
+   public interface IBatchStarterTask
+   {
+      StartableProcess StartValidation(string outputFolderPath);
```

Comment:
Is this the task that is going to also perform the comparison?
It feels that this should be more of a Runner like we have for the PArameterIdentification and this should return a Task that can be canceled using a CancelationToken.
The actual StartableProcess would not be returned